high_card.rb

```diff
@@ -1,5 +1,15 @@
 # Use these two arrays to generate a deck of cards.
 ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
+deck = []
+
+  ranks.each_with_index do |r, i|
+    suits.each do |suit|
+      deck.push({ ranks: r, suits: suit, })
```

Comment:
Looks like you are missing one more key-value pair in this hash, maybe a way to represent each card's value?